data/bootstrap_seed.yaml

```diff
@@ -0,0 +1,171 @@
+---
+webvirt: 192.168.1.1
+site_name: sitename
+maas_api_key: consumerkey:tokenkey:tokensecret
+maas_uuid: nonce
+host_profiles:
+- name: k8-node
+  hardware_profile: HPGen9v3
+  interfaces:
+  # Keyed on device_name
+  # pxe is a special marker indicating which device should be used for pxe boot
+  - device_name: pxe
+    network_link: pxe
+    slaves:
+    - prim_nic01
+    networks: 
+    - name: pxe
+  - device_name: bond0
+    network_link: gp
+    slaves:
+    - prim_nic01
+    - prim_nic02
+    networks:
+    - name: mgmt
+    - name: private
+  oob:
+    type: ipmi
+    network: oob
+    account: admin
+    credential: admin
+  storage:
+    boot_drive:
+      device: primary_boot
+      bootsize: 2g
+      rootsize: 50g
+  platform:
+    image: ubuntu_16.04_hwe
+    kernel_params: default
+  metadata:
+    introspection_url: http://172.16.1.10:9090
```

Comment:
In my mind if we have CIs that apply to all nodes without fail, we just have a top-level profile 'defaults'.